### ISSUE TITLE
Real time inserts of new commits

### ISSUE THREAD

**Issue by Teun** [Teun/git-flow-vis#32]
By regularly checking for new commits, we could have a live feel to the page. By requesting the branches feed, you can see in one request if new commits have been added to the repo.

**Comment by Teun** [comment]
Will implement by creating an extra API on GFV.branches. Host can provide updated or added branches, so the chart can be updated to reflect them.

**Comment by remie** [comment]
Shouldn't this mostly be implemented in the host? What kind of functionality should the library provide other than a "redraw" method?

**Comment by Teun** [comment]
Yes. Well. I'd like to have the host provide new or updated HEADs. The current implementation has only a full data call that must provide information about branches, tags and commits. Then, the component can request extra commits providing only a sha. I would like the host to be able to provide a set of branches (with ref and latestChangeset) . Then the chart can check if this was already known and if it needs to download extra commits. So a host only does the scheduled call for the current branches/HEADs. The logic to recognize changes to the current state can be done on the inside.
